src/status_im/native_module/impl/module.cljs

```diff
@@ -1,236 +0,0 @@
-(ns status-im.native-module.impl.module
```

Comment:
we needed this for tests right? how do tests work now?

Reply:
it was never used for tests, and there is only 2 functions that are not in an effect and hence called during tests, it is those whis (when (status)) guard